src/Tribe/Main.php

```diff
@@ -105,8 +108,8 @@ public function load_assets() {
 		tribe_assets(
 			$this,
 			array(
-				array( 'tribe-jquery-ui-theme', 'vendor/jquery/ui.theme.css' ),
-				array( 'tribe-clipboard', 'vendor/clipboard/clipboard.js' ),
+				array( 'tribe-jquery-ui-theme', $this->plugin_url . 'vendor/jquery/ui.theme.css' ),
```

Comment:
Whoops! That was actually from me debugging something and not meant to be in this PR. Good catch.